### ISSUE TITLE
Config not generating

### ISSUE THREAD

**Issue by A5ho9999** [ix0rai/rainglow#7]
Config file doesn't seem to be generating and the wiki is no help as what file type the config actually is. Same output on both Server and Client.

using version 1.0.0+mc1.19.3
```
config file not found or corrupted; failed to read: config will reset to default value
failed to read line "" of config file; line will reset to default value
could not write object "rainbow" of type string to config file under key "mode"!
could not write object ["blue", "white", "pink"] of type string list to config file under key "custom"!
could not write object true of type boolean to config file under key "enable_server_sync"!
```
```
could not write object ["blue", "red", "green", "pink", "yellow", "orange", "indigo", "purple", "white", "gray", "black"] of type string list to config file under key "custom"!
could not write object ["blue", "red", "green", "pink", "yellow", "orange", "indigo", "purple", "white", "gray", "black"] of type string list to config file under key "custom"!
could not write object ["blue", "red", "green", "pink", "yellow", "orange", "indigo", "purple", "white", "gray", "black"] of type string list to config file under key "custom"!
could not write object "all_colours" of type string to config file under key "mode"!
```

**Comment by ix0rai** [comment]
thank you for reporting this!
I've fixed it (the error was me being endlessly dumb and quite literally not having code to create a new config file if it didn't exist), and also added a blurb about where to find the file in the wiki

**Comment by ix0rai** [comment]
oh also: I have some other things I want to fix, but a 1.0.1 should come out today!

**Comment by ix0rai** [comment]
my development environment is having a lovely time exploding tonight, so I'm going to have to delay that update to tomorrow, sorry!

**Comment by A5ho9999** [comment]
That's fine, creating the file manually is easy enough now I know which type to use. Though sadly have to create a new issue for you.

**Comment by ix0rai** [comment]
this and some other issues have been fixed in 1.0.1, out on modrinth!
https://modrinth.com/mod/rainglow/version/1.0.1+mc1.19.3

**Comment by ix0rai** [comment]
@A5ho9999 ping :p
